binding/HarfBuzzSharp.Shared/Font.cs

```diff
@@ -17,11 +18,34 @@ public Font (Face face)
 			Handle = HarfBuzzApi.hb_font_create (face.Handle);
 		}
 
+		public Font (Font parent)
+			: this (IntPtr.Zero)
+		{
+			if (parent == null) {
+				throw new ArgumentNullException (nameof (parent));
+			}
+
+			if (parent.Handle == IntPtr.Zero) {
+				throw new ArgumentException (nameof (parent.Handle));
+			}
+
+			Handle = HarfBuzzApi.hb_font_create_sub_font (parent.Handle);
+		}
+
 		internal Font (IntPtr handle)
 			: base (handle)
 		{
 		}
 
+		public Font Parent {
+			get {
+				var parent = HarfBuzzApi.hb_font_get_parent (Handle);
```

Comment:
Introduce a backing field so we can store the .Net instance when a sub font is created by the binding